### ISSUE TITLE
Need to add alt text to zoom image for accessibility

### ISSUE THREAD

**Issue by arobbins** [willnguyen1312/zoom-image#259]
### Description of the problem

While running my site through the [axe DevTool](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?pli=1), I received errors relating to missing alt text.

After investigating, it appears the dynamically generated image from `zoom-image` does not contain an alt attribute. See attachment below.

Can you provide an option to pass in a custom alt text to these dynamically generated images?

<img width="1079" alt="Screenshot 2024-01-26 at 12 04 04 PM" src="https://github.com/willnguyen1312/zoom-image/assets/961947/5a49bc9f-115a-4f0d-b604-e6accf89d9e4">

**Comment by willnguyen1312** [comment]
Hi @arobbins, sorry for taking so long for replying on this issue as I was occupied at work. I am going to investigate this and get back to you soon :)

**Comment by willnguyen1312** [comment]
Hi there, I updated the API so that now we can pass some properties to the generated image - https://willnguyen1312.github.io/zoom-image/api/createZoomImageHover.html#type-declaration. Please feel free to let me know if something does not work as expected. Thanks :)

**Comment by arobbins** [comment]
Thank you Nam!
